app/models/peer_review.rb

```diff
@@ -8,9 +10,21 @@ class PeerReview < ActiveRecord::Base
   validates_presence_of :result
   validates_numericality_of :reviewer_id, only_integer: true, greater_than: 0
   validates_numericality_of :result_id, only_integer: true, greater_than: 0
+  validate :no_students_should_be_reviewer_and_reviewee
 
   def reviewee
     # TODO - Research optimizing or see if rails can do better
     Grouping.joins({ submissions: { results: :peer_reviews }}).where('peer_reviews.id = ?', self.id).first
   end
+
+  def no_students_should_be_reviewer_and_reviewee
+    student_id_set = Set.new
+    reviewer.students.each { |student| student_id_set.add(student.id) }
+    result.submission.grouping.students.each do |student|
+      if student_id_set.include?(student.id)
+        errors.add(:reviewer_id, I18n.t('peer_reviews.cannot_allow_reviewer_to_be_reviewee'))
```

Comment:
I want to remove the I18n but it errors out without it, is there a way to include the file somehow? I don't understand how the `results_controller.rb` can get away with it but this controller can't.